client/src/components/DisplayTable/DisplayTable.js

```diff
@@ -490,7 +556,11 @@ export class SmartDisplayTable extends React.Component {
       })
     );
     return (
-      <DisplayTable {...this.props} column_configs={smart_column_configs} />
+      <DisplayTable
+        {...this.props}
+        column_configs={smart_column_configs}
+        show_pagination_controls={_.size(data) > 100}
```

Comment:
Small DRY point, could be `> DisplayTable.defaultProps.page_size_increment`

Reply:
TIL 😮🙏